app/src/main/java/com/fpvout/digiview/tutorial/DroneFragment.java

```diff
@@ -0,0 +1,66 @@
+package com.fpvout.digiview.tutorial;
+
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.fpvout.digiview.R;
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Use the {@link DroneFragment#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class DroneFragment extends Fragment {
+
+    // TODO: Rename parameter arguments, choose names that match
+    // the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+    private static final String ARG_PARAM1 = "param1";
```

Comment:
Sorry if this is for some Android magic that I'm not aware of (since I'm not an Android dev,) but I don't see where these parameters are used. `mParam1` and 2 are set, but is that just boilerplate that should be removed?

Reply:
You're completely right. That is some template that I forgot to delete...

Reply:
Fixed in facbbdaa5bc72adb6b2e2dc57f8230d9749e4f09